.github/ISSUE_TEMPLATE/operate-feature.md

```diff
@@ -6,38 +6,33 @@ labels: ["kind/feature", "component/operate"]
 assignees: ''
 
 ---
-<!-- Remember to link this feature issue in the parent Epic -->
 
-### Description
-<!-- [Optional] A clear description of the story -->
+### Description (required on creation)
+
+<!-- A clear description of the feature -->
+
+### User story (Required on creation)
 
-### User Story
-<!-- [Mandatory field] -->
 ```Gherkin
 Scenario: As an Operate user
     Given I am logged in
     When I select one item/tab in the toolbar
     Then this item should be shown in highlighted (in bold)
 ```
+
 ### Designs
+
 <!-- [Optional] <Figma Link> -->
 
-### Acceptance Criteria
-<!-- [Mandatory field] The assignee will fill the Acceptance Criteria. -->
-- [ ]
+### Functional Requirements (required before implementation)
 
-### Definition of Ready - Checklist
-<!-- the assignee will check the DOR. -->
+<!-- Requirements that directly implement the user story -->
+ 
+### Technical Requirements (required before implementation)
 
-- [ ] The issue has a meaningful title, description, and testable acceptance criteria
-- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
-- [ ] If HELM charts need to be updated, an issue is created in the [camunda-platform-heml](https://github.com/camunda/camunda-platform-helm) repo, and the issue is added to our Operate project board.
-- [ ] Potential Security risks have been considered and added to the product-hub issue risk assessment
-- [ ] Cross-team dependencies have been considered
+<!-- Requirements that enable the user story to be implemented (e.g. technical foundations) -->
 
-Optional:
-- [ ] Design input has been collected by the assignee
-### :point_right: Handover Dev to QA 
+### Handover Dev to QA (required before manual testing)
```

Comment:
@ArezooAzar can you please review the contents of this section? It is fairly long (and thus has a similar tendency to be seen as bloat in the ticket description that people will ignore after some time). Can we make it more compact? Similar for the bug report template.

Reply:
Here's what I need to know: 
Where to test the PR ( environment or link to that) 
Release version for this PR 
If there is any test date that can be used to test the PR ( like any process in demo test data) 
if any of this information is already provided in the template. please let me know where can i find it so we can avoid duplication here.

Reply:
Thanks, I will adjust the template accordingly so that it becomes a bit more compact but still contains what you have shared.